server/target/target.go

```diff
@@ -36,7 +43,18 @@ const (
 	coverageCommand = "coverage"
 
 	// The number of distinct commits returned in GetTargetHistoryResponse.
-	targetPageSize = 20
+	targetHistoryPageSize = 20
+
+	// The number of targets returned in the GetTarget response.
```

Comment:
nit: in a single page of the GetTarget response.

Reply:
Done